tests/node_pool_deployer.py

```diff
@@ -26,8 +27,9 @@ def __init__(self, testconfig):
         self.pool_name = f"pool-{self.namespace}"
         self.cluster_name = "spacemesh-cluster-elk"
         self.zone = "us-west1-a"
+        self.project_name = "spacemesh-198810"
```

Comment:
I still think this (cluster_name, zone, project_name) should not be hardcoded, can we move it into the env?

Reply:
are cluster_name and zone ok or should I remove those as well?

Reply:
fixed